sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -217,9 +217,49 @@ export namespace ConnectionContext {
       }
     };
 
+    const protocolError: OnAmqpEvent = async (context: EventContext) => {
+      const connectionError = context.connection && context.connection.error ? context.connection.error : undefined;
```

Comment:
`undefined` is the natural state of things. So this can be simplified as below
```suggestion
      const connectionError = context.connection && context.connection.error;
```